classes/ActionScheduler_wpPostStore.php

```diff
@@ -317,9 +321,12 @@ public function query_actions( $query = array() ) {
 		}
 		$sql .= " WHERE post_type=%s";
 		$sql_params[] = self::POST_TYPE;
-		if ( $query['hook'] ) {
+		if ( $query['hook'] && is_scalar( $query['hook'] ) ) {
 			$sql .= " AND p.post_title=%s";
 			$sql_params[] = $query['hook'];
+		} else if ( $query['hook'] ) {
+			$sql .= ' AND p.post_title IN (' . implode( ',', array_fill( 0, count( $query['hook'] ), '%s' ) ) . ')';
+			$sql_params = array_merge( $sql_params, $query['hook'] );
```

Comment:
This duplication highlights issues with duplicating code in `query_actions_count()` from `query_actions()`. That's beyond the socpe of the code here though, so I've mentioned it here: https://github.com/Prospress/action-scheduler/pull/88/#issuecomment-347346308